app/assets/stylesheets/rent.scss

```diff
@@ -5,3 +5,10 @@
   height:auto;
   min-height: 30%;
 }
+span, li {
+  #overlapping {
+    color: #a94442;
```

Comment:
Color literals like `#a94442` should only be used in variable declarations; they should be referred to via variable everywhere else.